test/Benchmarks/src/Table/Cross_Tab.enso

```diff
@@ -0,0 +1,51 @@
+from Standard.Base import all
+
+from Standard.Table import Table
+from Standard.Table.Data.Aggregate_Column.Aggregate_Column import Count, Sum
+
+from Standard.Test import Bench, Faker
+
+## Bench Utilities ============================================================
+
+num_iterations = 10
+seed=1646322139
+
+create_table : Table
+create_table num_rows =
+    faker = Faker.new seed
+    sizes = ['XXS', 'XS', 'S', 'M', 'L', 'XL', 'XXL']
+    stores = ["Brixton", "Chelsea", "Fulham", "Mayfair", "Notting Hill", "Peckham", "Richmond", "Westminster", "Whitechapel", "Wimbledon"]
+    types = ["Gravel", "Mountain", "Road"]
+    vgen v = 0.up_to num_rows . map _-> faker.vector_item v
+
+    size = ["size", vgen sizes]
+    store = ["store", vgen stores]
+    typ = ["type", vgen types]
+    price = ["price", 0.up_to num_rows . map _-> faker.decimal -200.0 200.0]
+
+    Table.new [size, store, typ, price]
+
+# The Benchmarks ==============================================================
+bench =
+    counts = [[25, 1000], [250, 1000], [2500, 100], [25000, 10]]
+    counts.map c->
+        num_rows = c.at 0
+        iter_size = c.at 1
+
+        table = create_table num_rows
+
+        measure group_by name_column values=[Count] =
+            operation = num_rows.to_text + "-" + (group_by.join '+') + "-" + name_column + "-" + (values.map .to_text . join "+")
+            approach = "one index"
+            name = operation + "/" + approach
+            Bench.measure (table.cross_tab0 group_by name_column values) name iter_size num_iterations
```

Comment:
Did you check this benchmark on the latest version? I don't think identifier `cross_tab0` exists.

Please make sure the benchmarks are runnable before they are merged.

Reply:
Also, ideally we should use the group builder pattern that should be available in `develop` by now.

Reply:
Done and done.